c++/src/ColumnWriter.cc

```diff
@@ -1453,7 +1453,7 @@ namespace orc {
                      uint64_t numValues) override;
 
   private:
-    char buffer[16];
+    char buffer[20];
```

Comment:
19 should be sufficient, right?

Reply:
Yes, both buffers are "aligned" to 2 bytes